traitsui/wx/color_trait.py

```diff
@@ -113,8 +115,8 @@ def convert_to_color(object, name, value):
     if isinstance(value, tuple):
         return tuple_to_wxcolor(value)
 
-    elif isinstance(value, ColourPtr):
-        return wx.Colour(value.Red(), value.Green(), value.Blue())
+    elif isinstance(value, PyfaceColor):
+        return value.to_toolkit()
 
     elif isinstance(value, wx.Colour):
         return value
```

Comment:
do WxColor not handle spaces (I'm assuming not since there is no wx test analogous to the qt `test_name_string_with_space`) / if so why can't they?
Can we not just add a `value = value.replace(" ", "")` in the `elif isinstance(value, str):` block below?


Similarly I notice we still use `eval` rather than `literal_eval` on line 136.  Same question here as above

Reply:
I've made these updates, and added some tests around this and what happens if eval doesn't give something reasonable in several different ways.